packages/vkui/src/components/Spinner/Spinner.module.css

```diff
@@ -20,14 +20,16 @@
  * PanelHeader
  */
 
```

Comment:
```suggestion
 * CMP:
 * PanelHeader
 */
```

Чтобы к единому формату привести, раз все равно рядышком влезли. 🙈

Reply:
done